app/namespace.go

```diff
@@ -588,6 +612,30 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				return c.createNamespace(n, unp)
 			},
 		},
+		{
+			Name:    "add-region",
+			Usage:   "Add an new region to the temporal namespace",
+			Aliases: []string{"c"},
+			Flags: []cli.Flag{
+				RequestIDFlag,
+				ResourceVersionFlag,
+				&cli.StringFlag{
```

Comment:
[nit] isn't this defined somewhere already  where we could reuse?

Reply:
No in the repo right now.